demisto_sdk/commands/doc_reviewer/doc_reviewer.py

```diff
@@ -319,6 +319,8 @@ def check_word(self, word):
         word = self.remove_punctuation(word)
         sub_words = []
         if '-' in word:
+            if not self.spellchecker.unknown([word]):
```

Comment:
shouldnt we split by - and check each?

Reply:
that's exactly the issue..
because we split the command name by '-' we get the error..
if we first check to ensure it's not a command name then there's a point to split it.

Reply:
not sure I understand. Lets discuss